thoth/storages/data/alembic/versions/ecb0ca258835_flag_for_if_specific_hash_has_gone_.py

```diff
@@ -0,0 +1,47 @@
+"""Flag for if specific hash has gone missing, defaults to present
+
+Revision ID: ecb0ca258835
+Revises: 0ea53ec16d6b
+Create Date: 2020-03-26 15:26:00.311857+00:00
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'ecb0ca258835'
+down_revision = '0ea53ec16d6b'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('python_artifact', sa.Column('present', sa.Boolean(), nullable=False, server_default="false"))
+
+
+    # NOTE: I am unsure where these came from?
```

Comment:
This is happening because you are changing the datatype of `artifact_hash_sha256` and `artifact_name` from Text to String, why that? We want to have Text and do not use String actually.